net.agalwood.Motrix.metainfo.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>net.agalwood.Motrix</id>
+
+  <name>Motrix</name>
+  <summary>A full-featured download manager</summary>
+
+  <developer_name>AGALWOOD</developer_name>
+
+  <metadata_license>MIT</metadata_license>
```

Comment:
CC0 as a license for metadata is preferred. Since it's here I assume you created the file.

Reply:
Typo. It should be the `<project_license>`